### DISCUSSION TITLE
How to apply node affinities ,taints and tolerations Strimzi Pods

### DISCUSSION THREAD

**Discussion by iamrsaravana** [strimzi/strimzi-kafka-operator#5109] [Q&A]
We are planning to deploy Stimizi kafka in our kubernetes cluster. we have implemented taints and tolerance for our existing application to make sure those pods will be used particular nodes in our cluster. i could not find configuration for Strimzi kafka. 

if any body implemented these feature, could you explain how to apply Strimzi kafka or links

**Comment by scholzj** [comment]
Please check the docs: https://strimzi.io/docs/operators/latest/full/using.html#assembly-scheduling-str
